qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -347,6 +414,158 @@ public void test92ElasticsearchNodeCliPackaging() {
         final Installation.Executables bin = installation.executables();
 
         final Result result = sh.run(bin.elasticsearchNode + " -h");
-        assertThat(result.stdout, containsString("A CLI tool to do unsafe cluster and index manipulations on current node"));
+        assertThat(
+            "Failed to find expected message about the elasticsearch-node CLI tool",
+            result.stdout,
+            containsString("A CLI tool to do unsafe cluster and index manipulations on current node")
+        );
+    }
+
+    /**
+     * Check that no core dumps have been accidentally included in the Docker image.
+     */
+    public void test100NoCoreFilesInImage() {
```

Comment:
This numbering will not work unless all the test numbers less than 100 are padded to be 3 digits long.